.github/renovate.json5

```diff
@@ -8,7 +8,7 @@
   "prConcurrentLimit": 5,
 // The branches renovate should target
 // PLEASE UPDATE THIS WHEN RELEASING.
-  "baseBranches": ["main","release-1.21","release-1.22"],
+  "baseBranches": ["main","release-1.21","release-1.22", "release-1.23"],
```

Comment:
Didn't we say we only support the latest minor? Should we remove 1.21 and 1.22 then?

Reply:
Not yet. We still need to support 1.21 for a few weeks